doc/rpi_guide.md

```diff
@@ -0,0 +1,232 @@
+# OpenDashboard - Raspberry Pi Guide
```

Comment:
just like the cool kids!

Do note that afaik the cpu is a bit diff on the rpi4.. idk for sure